### DISCUSSION TITLE
Is it possible to add documents together?

### DISCUSSION THREAD

**Discussion by dlampa** [mikefarah/yq#806] [Q&A] [answered]
For example:

file1.yaml:
```
name: "file1"
```

file2.yaml:
```
name: "file2"
```

After combining to yield:
```
name: "file1"
---
name: "file2"
```

Thanks!

**Comment by mikefarah** [comment, marked as answer]
yep:

```
yq e '.' file1.yml file2.yml
```

**Reply by dlampa** [reply]
Great, thanks! And could one do the same by passing one of the files through STDIN?

**Reply by dlampa** [reply]
To answer my own question - yes:
`yq -n e '.name="stdin"' | yq e '.' file1.yaml -`
